conventions/code-reviews.md

```diff
@@ -0,0 +1,74 @@
+# Code Reviews
+
+### The purposes of code review at Shift3 (and in the industry generally) are:
+1. Teachability:
+    * They help you remain teachable
+    * They allow us to learn from each other's experience. What developer _doesn't_ want to keep learning?
+2. Catch Bugs:
+    * They catch potential bugs from being integrated into the main code base
+    * They ensure code quality and veracity
+3. Improve Code:
+    * Improve your code base by having another set of eyes on it
+    * Catch security vulnerabilities that you may not have known about
+    * Give more developers the ability to move between projects without the initial shock of not knowing _anything_ about the code
+
+## Process
+
+### Author
+* At Shift3 we have a simple code review process
+
+1. Before you create your Pull Request or tag anyone in anything, double check that you're ready
+
+    Pre-PR checklist:   
+    - [ ] Test the specific code you're submitting
+    - [ ] Use a linter (esLint is probably going to be the standard) to take care of whitespace and styling
+    - [ ] Remove your console references and debuggers
+    - [ ] Make sure your code changes are uner 500 lines of code (break it up into more than 1 PR if it exceeds this)
```

Comment:
console references and debuggers love it

Reply:
oh that's much better than what I was thinking!